### ISSUE TITLE
Always notify

### ISSUE THREAD

**Issue by moaeddy** [AdvancedHacker101/Bypass-Uac#1]
can this work if UAC set to ALWAYS NOTIFY?

**Comment by fxrstor** [comment]
Nope, it won't work If UAC Level is 4 aka Always Notify, because UAC Level 4 wipes out auto elevation from system executables.
If you execute Dism, printui, or any auto elevated executable on UAC Level 4, then it will show the UAC prompt to user.

I request you to close this issue as your question is answered!
Please close this issue.
